### PR TITLE
Chore: Updates CHANGELOG.md for 7.0.3

### PR DESCRIPTION
**Pull request by mckn**
./node_modules/.bin/grafana-toolkit changelog -m 179       
Running in local/linked mode
⠸ Generating changelog  
  
  /*** Grafana changelog  ***/
  
  ### Features / Enhancements
  * **Dashboard**: Get panels in collapsed rows. [#25079](https://github.com/grafana/grafana/pull/25079), [@peterholmberg](https://github.com/peterholmberg)
  * **Stats**: include all fields. [#24829](https://github.com/grafana/grafana/pull/24829), [@ryantxu](https://github.com/ryantxu)
  * **Variables**: change VariableEditorList row action Icon to IconButton. [#25217](https://github.com/grafana/grafana/pull/25217), [@hshoff](https://github.com/hshoff)
  
  ### Bug Fixes
  * **Cloudwatch**: Fix dimensions of DDoSProtection. [#25317](https://github.com/grafana/grafana/pull/25317), [@papagian](https://github.com/papagian)
  * **Configuration**: Fix env var override of sections containing hyphen. [#25178](https://github.com/grafana/grafana/pull/25178), [@marefr](https://github.com/marefr)
  * **Do not show alerts tab when alerting is disabled**. [#25285](https://github.com/grafana/grafana/pull/25285), [@dprokop](https://github.com/dprokop)
  * **Jaeger**: fixes cascader option label duration value. [#25129](https://github.com/grafana/grafana/pull/25129), [@Estrax](https://github.com/Estrax)
  * **Transformations**: Fixed Transform tab crash & no update after adding first transform. [#25152](https://github.com/grafana/grafana/pull/25152), [@torkelo](https://github.com/torkelo)
  
✔ Generating changelog

### CODE REVIEW

**Review by papagian — CHANGES_REQUESTED** [review]
I think that this should appear under bug fixes:
`Dashboard: Get panels in collapsed rows. #25079, @peterholmberg`

Probably that's because the closed issue has `7.0.1` milestone instead of `7.0.3`

**Comment by mckn** [comment]
> I think that this should appear under bug fixes:
> `Dashboard: Get panels in collapsed rows. #25079, @peterholmberg`
> 
> Probably that's because the closed issue has `7.0.1` milestone instead of `7.0.3`

Awesome feedback! I have updated the CHANGELOG.md, the community post and the Github release notes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+### Features / Enhancements
+* **Stats**: include all fields. [#24829](https://github.com/grafana/grafana/pull/24829), [@ryantxu](https://github.com/ryantxu)
+* **Variables**: change VariableEditorList row action Icon to IconButton. [#25217](https://github.com/grafana/grafana/pull/25217), [@hshoff](https://github.com/hshoff)
+
+### Bug Fixes
+* **Dashboard**: Get panels in collapsed rows. [#25079](https://github.com/grafana/grafana/pull/25079), [@peterholmberg](https://github.com/peterholmberg)
+* **Cloudwatch**: Fix dimensions of DDoSProtection. [#25317](https://github.com/grafana/grafana/pull/25317), [@papagian](https://github.com/papagian)
+* **Configuration**: Fix env var override of sections containing hyphen. [#25178](https://github.com/grafana/grafana/pull/25178), [@marefr](https://github.com/marefr)
+* **Do not show alerts tab when alerting is disabled**. [#25285](https://github.com/grafana/grafana/pull/25285), [@dprokop](https://github.com/dprokop)
```
Comment (papagian) [inline CHANGELOG.md:11]:
```suggestion
* **Cloudwatch**: Fix dimensions of DDoSProtection. [#25317](https://github.com/grafana/grafana/pull/25317), [@papagian](https://github.com/papagian)
* **Configuration**: Fix env var override of sections containing hyphen. [#25178](https://github.com/grafana/grafana/pull/25178), [@marefr](https://github.com/marefr)
* **Dashboard**: Get panels in collapsed rows. [#25079](https://github.com/grafana/grafana/pull/25079), [@peterholmberg](https://github.com/peterholmberg)
* **Do not show alerts tab when alerting is disabled**. [#25285](https://github.com/grafana/grafana/pull/25285), [@dprokop](https://github.com/dprokop)
```

Comment (papagian) [reply CHANGELOG.md:11]: I think they are in alphabetical order

**Review by papagian — APPROVED** [review]

**Comment by Zayan1221** [comment]
Hello,
yum install grafana-7.0.3-1.x86_64.rpm
Loaded plugins: fastestmirror
Examining grafana-7.0.3-1.x86_64.rpm: grafana-7.0.3-1.x86_64
Cannot install package grafana-7.0.3-1.x86_64. It is obsoleted by installed package grafana-enterprise-7.0.2-1.x86_64
Error: Nothing to do

**Comment by Zayan1221** [comment]
This is solution - It has been resolved now:
yum erase grafana-enterprise
wget https://dl.grafana.com/oss/release/grafana-7.0.3-1.x86_64.rpm
yum install grafana-7.0.3-1.x86_64.rpm
service grafana-server status
systemctl daemon-reload
service grafana-server restart
# grafana-cli -v
Grafana CLI version 7.0.3

**Comment by Zayan1221** [comment]
But the initial problem still exist
it shows 7.0.2 on Grafana and Edit option still doesnot shows

**Pull request merged**